### PR TITLE
Scaffold additional qs

### PR DESCRIPTION
**Pull request by analoo**

### CODE REVIEW

**Comment by bseeger** [comment]
Screenshot 2023-12-15 at 10 14 04 AM

Something about this PR is truncating the distance between the "card" and the footer.

File: src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java [hunk 1]
```diff
@@ -123,11 +123,34 @@ public class LaDigitalAssister extends FlowInputs {
...
+//  @DynamicField
+  private String pregnancyDueDateDay;
+
+//  @NotBlank(message="{error.missing-general}")
+//  @DynamicField
+  private String pregnancyDueDateMonth;
+
+//  @NotBlank(message="{error.missing-general}")
+//  @DynamicField
+  private String pregnancyDueDateYear;
```
Comment (bseeger) [inline src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java:153]: I'm noticing that the field names used in the templates don't align with what is here, and maybe that is why they are not working?  `name="pregnancyDueDate_wildcard_9384801b-95ab-4907-aab6-376b634d4a78Year"`

File: src/main/resources/templates/laDigitalAssister/householdPregnancyDueDate.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          th:replace="'fragments/cardHeader' :: cardHeader(header=#{pregnantduedate.header}, subtext=#{pregnantduedate.subheader})"/>
+        <th:block th:replace="'fragments/form' :: form(action=${formAction}, content=~{::formContent})">
+          <th:block th:ref="formContent">
+
+            <div class="form-card__content">
+              <div th:each="individual, iter: ${inputData.get('pregnanciesFullNames[]')}" class="spacing-below-15">
+                <hr>
+                <th:block th:with="pregnancyDueDateInputName=${'pregnancyDueDate' + T(formflow.library.inputs.FieldNameMarkers).DYNAMIC_FIELD_MARKER + individual.uuid}">
+                  <th:block th:replace="~{fragments/inputs/date ::
+                    date(inputName=${pregnancyDueDateInputName},
```
Comment (bseeger) [inline src/main/resources/templates/laDigitalAssister/householdPregnancyDueDate.html:22]: Oh, I see... this is a little challenging.   Maybe we need to change the date fragment to work with a "suffix" to the name (where the "uuid" is the suffix passed in).  Hmmm...

File: src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java (hunk 1 above)
Comment (bseeger) [reply src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java:153]: I see why below.  The fragments are definitely not prepare to work with dynamic fields, but I think we can definitely tweak them to fit our needs.

File: src/main/resources/templates/laDigitalAssister/householdPregnancyDueDate.html (hunk 2 above)
Comment (bseeger) [reply src/main/resources/templates/laDigitalAssister/householdPregnancyDueDate.html:22]: Maybe we could override the template and have the call be: 

```
${pregnancyDueDateInputName} would equal "pregnancyDueDate"
${dynamicFieldSuffix} would equal `T(formflow.library.inputs.FieldNameMarkers).DYNAMIC_FIELD_MARKER + individual.uuid`

date(inputName=${pregnancyDueDateInputName},
         inputNameSuffix = ${dynamicFIeldSuffix},   // optional field
         label = #{...},
         groupName='pregnancyDueDate'}
```

Comment (analoo) [reply src/main/resources/templates/laDigitalAssister/householdPregnancyDueDate.html:22]: @bseeger this is really helpful! Thank you

**Pull request closed without merging**